Assets/Scripts/Controllers/BuildModeController.cs

```diff
@@ -190,5 +193,21 @@ public void DoBuild(Tile t)
 
     }
 
+    public bool BuildJobWouldOverlapExistingBuildJob(Tile t, string furnitureType)
```

Comment:
@bjubes That is exactly what i was going for, but i could not get the wording right. Thank you.